packages/server-wallet/src/message-service/test-message-service.ts

```diff
@@ -44,20 +47,80 @@ export class TestMessageService implements MessageServiceInterface {
    * @returns
    */
   protected constructor(handleMessage: MessageHandler, protected _logger?: Logger) {
+    super();
     this._options = {dropRate: 0, meanDelay: undefined};
     this._abortController = new AbortController();
     this._handleMessages = async messages => {
       for (const message of messages) {
-        // This prevents triggering messages after the service is destroyed
-        if (!this._destroyed) return handleMessage(message, this);
+        if (!this._destroyed) {
+          await handleMessage(message);
+        }
       }
     };
   }
 
-  static async create(
-    incomingMessageHandler: MessageHandler,
+  static setLatencyOptions(peerWallets: TestPeerWallets, options: Partial<LatencyOptions>): void {
```

Comment:
Should these `static` functions just be normal functions defined outside of the class? Any benefit to having these as `static` functions?

Reply:
I think my hope was to aid in discoverability. Since these functions are pretty important to the `TestMessageService` I wanted to make it easy to find them. However they're reliance on `TestPeerWallets` might make them better suited to live somewhere else.